spec/src/main/asciidoc/preface.asciidoc

```diff
@@ -35,9 +35,17 @@ This document is organized in 4 parts:
 
 ==== Jakarta Contexts and Dependency Injection 4.1
 
-link:https://jakarta.ee/specifications/cdi/4.1/[CDI 4.1] deprecates the Unified EL integration API in `BeanManager` and places the relevant methods to a dedicated interface `ELAwareBeanManager`, which is present in a new supplemental API artifact: `jakarta.enterprise:jakarta.enterprise.cdi-el-api`.
+link:https://jakarta.ee/specifications/cdi/4.1/[CDI 4.1] no longer specifies integration with Jakarta EE. This will now be specified in the Jakarta EE platform specification.
```

Comment:
```suggestion
link:https://jakarta.ee/specifications/cdi/4.1/[CDI 4.1] no longer specifies integration with Jakarta EE. This will now be specified in the Jakarta EE platform/web profile/core profile specification.
```